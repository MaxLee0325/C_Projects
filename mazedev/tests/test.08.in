Alpha
Bravo
Charlie
Delta
Echo
Foxtrot
END
Alpha Bravo
Bravo Charlie
Charlie Delta
Delta Echo
Echo Foxtrot
END
Alpha
Bravo
Charlie
Delta
Echo
Foxtrot
END
