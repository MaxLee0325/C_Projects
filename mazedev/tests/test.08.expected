Entering room Alpha.
Entering room Bravo.
Entering room Charlie.
Entering room Delta.
Entering room Echo.
Entering room Foxtrot.
Ended in room Foxtrot.
